### PR TITLE
Check if XFS is in use and influence PrepareUpgradeTransaction

### PR DESCRIPTION
**Pull request by artmello**
Based on the information of XFS being in use without a ftype, create or not disk image inside PrepareUpgradeTransaction actor

### CODE REVIEW

File: repos/system_upgrade/el7toel8/actors/checkxfs/libraries/library.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+    mountpoints = fstab_data | mount_data | systemdmount_data
+    
+    xfs_presence = XFSPresence()
+    # By now, we only care for XFS without ftype in use for /var
+    for mp in ('/var', '/'):
+        if mp in mountpoints:
+            xfs_presence.present = True
+            xfs_presence.without_ftype = is_xfs_without_ftype(mp)
+            break
```
Comment (jmikovic) [inline repos/system_upgrade/el7toel8/actors/checkxfs/libraries/library.py]: couldn't this be a problem if both /var and / use xfs but /var have ftype=1 and / ftype=0, as diskimage wouldn't be created in that case

**Comment by pirat89** [comment]
@jmikovic: sorry. I closed your comment by mistake. Github is stucked by ... whatever and I cannot open it, show it, ... just anything right now.

File: repos/system_upgrade/el7toel8/actors/checkxfs/libraries/library.py (hunk 1 above)
Comment (artmello) [reply repos/system_upgrade/el7toel8/actors/checkxfs/libraries/library.py]: @jmikovic thanks for reviewing. I tried to change the approach now to notify other actors if either / or /var has XFS without ftype. Let me know if this handles your concern.

**Comment by artmello** [comment]
@pirat89 / @jmikovic Is the "closed comment" the one about /var with ftype and / without? Or something else that needs to be handled?

**Comment by jmikovic** [comment]
@artmello it was the one about /var, I just reopened it

**Pull request merged**